mlir/test/lib/Dialect/Test/TestPatterns.cpp

```diff
@@ -470,34 +527,6 @@ struct TestStrictPatternDriver
     }
   };
 
-  // Replace an operation may introduce the re-visiting of its users.
-  class ReplaceWithNewOp : public RewritePattern {
```

Comment:
Why did you need all this code motion?

Reply:
This was originally defined inside a test pass class and I wanted to use it in my new test pass